### ISSUE TITLE
OnePlus Two Launching Error 64-bit

### ISSUE THREAD

**Issue by jakey1995abc** [zhuowei/Boardwalk#238]
Version of Boardwalk:1.1

Version of Android:5.1.1

Device model:ONE A2003 (OnePlus Two)

Description of the problem: After selecting Minecraft version and launching, it stops installing with the error in the image. The OnePlus Two is a 64bit phone running oxygenOS 2.1.2 with root and busybox installed.
![screenshot_2016-01-04-14-44-02](https://cloud.githubusercontent.com/assets/7613005/12091964/33417912-b2f2-11e5-9b9a-6bb3cd16b181.png)

**Comment by liushuyu** [comment]
Yeah, I encountered the same problem on my One Plus 2 with extractly the same output

**Comment by MichaelSoftware** [comment]
Like ive been meantioning Boardwalk is not working on 1.1 on any android device besides 5.1.1 and even some android 5.1 devices are not able to run. Install Boardwalk 1.0.7 , it should be fixed.

**Comment by liushuyu** [comment]
>Install Boardwalk 1.0.7 , it should be fixed.

Where can I get this early version?

**Comment by jbdman** [comment]
Ill try and put a link.

**Comment by jbdman** [comment]
Here is said link: https://github.com/zhuowei/Boardwalk/blob/5c543ffb3243b9cae595d8f72997110490d70095/Boardwalk-release.apk

**Comment by liushuyu** [comment]
>Here is said link: >https://github.com/zhuowei/Boardwalk/blob/5c543ffb3243b9cae595d8f72997110490d70095/Boardwalk-release.apk

This still didn't work though, the error message is the same.

**Comment by zhuowei** [comment]
wat
1.1 and 1.2 beta works fine on Android 6.0. http://zhuoweizhang.net/Boardwalk/Boardwalk-release.apk @jbdman

**Comment by liushuyu** [comment]
Ah, yes , because your are using Android M, but mine is Android L.
2016年3月22日 08:28，"zhuowei" <notifications@github.com>写道：

**Comment by zhuowei** [comment]
@liushuyu I don't have a 64-bit device running 5.1. I'll take a look.

**Comment by liushuyu** [comment]
I seem to know the problem, the Oxygen and Hydrogen OS which comes with Oneplus 2 have incomplete libc support. Another software Termux detects that Oneplus 2 is arm32 device while it is surely a 64-bit device but lacks many functions in libc.

**Comment by MichaelSoftware** [comment]
@zhuowei If u need a device running a 64 bit of M. I'd suggest looking at running a Android VM from Android Studio. I ran a VM Android 6.0 on it. Let me know how it goes. 

And as for your problem @liushuyu  i did look at My 1P1 and it does not have comple libc support just like the 2 doesnt. I am rn working on fixing this.

**Comment by MichaelSoftware** [comment]
It doesnt matter on the OS. Somehow 5.1.1 isnt working anymore. I used to
be able to play it on my S6 Edge and now it doesnt work anymore..

On Fri, Mar 25, 2016 at 9:38 AM, liushuyu <notifications@github.com> wrote:

> I seem to know the problem, the Oxygen and Hydrogen OS which comes with
> Oneplus 2 have incomplete libc support. Another software Termux detects
> that Oneplus 2 is arm32 device while it is surely a 64-bit device but lacks
> many functions in libc.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/zhuowei/Boardwalk/issues/238#issuecomment-201281474>
>



-- 
Problems with the email?

Please email me at

michaelwojcik2@gmail.com
